build/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:focal
+WORKDIR /trees
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install build tools.
+RUN apt-get update && \
+    apt-get --no-install-recommends install -y \
+	apt-transport-https \
+	apt-utils \
+	build-essential \
+	cmake \
+	ca-certificates \
+	git \
+	ninja-build \
+	python \
+	python3 \
+	xorg-dev \
+	clang-7 \
```

Comment:
Could we use clang-9+ if we can choose our own Ubuntu? We were restricted to 7 because of the ancient Ubuntu in CI (14 or 16)

Reply:
Yes, this is something we can upgrade after we move to a Docker-based workflow for our Linux builds.